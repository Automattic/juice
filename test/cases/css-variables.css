:root{
	--border-width:1px;
	--border-color:red;
	--border-style-default:solid;
}
p {
  --border-width:2px;
  --border-style:var(--border-style-default);
  border: VAR(--border-width) var(--border-style) var(--border-color);
  background: var(--bg);
}

td{
  background-color: var(--bg,var(--border-color,rgb(255,0,0)));
}
