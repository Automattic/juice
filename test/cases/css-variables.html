
<html>
<body>
<p>woot</p>
<td>woot</td>
</body></html>
