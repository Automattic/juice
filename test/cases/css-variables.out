<html>
<body>
<p style="border: 2px solid red; background: var(--bg);">woot</p>
<td style="background-color: red;" bgcolor="red">woot</td>
</body></html>
